ckb-bin/src/subcommand/init.rs

```diff
@@ -24,6 +24,11 @@ pub fn init(args: InitArgs) -> Result<(), ExitCode> {
         return Ok(());
     }
 
+    if args.chain != "dev" && (args.use_default_spec || !args.customize_spec.is_unset()) {
+        eprintln!("Customizing consensus parameters for chain spec only works for dev chains.");
+        return Err(ExitCode::Failure);
```

Comment:
First, unifying the `ExitCode` isn't target of this PR.
I just follow the the usual practice under `subcommand` directory.

Second, I think `ExitCode::Config` is incorrect. It's not a configuration error.
And it's not `ExitCode::Cli` because the arguments are right.
For example, if the input timestamp couldn't parse to a number, the `ExitCode::Cli` will be returned.
https://github.com/nervosnetwork/ckb/blob/80a40a5c58f1dba47ea4fdbf28702784984bdf9b/util/app-config/src/lib.rs#L239-L250

Here is a runtime error.

Reply:
Got it.